tests/rest/admin/test_server_notice.py

```diff
@@ -415,6 +429,110 @@ def test_send_server_notice_delete_room(self) -> None:
         # second room has new ID
         self.assertNotEqual(first_room_id, second_room_id)
 
+    @override_config(
+        {
+            "server_notices": {
+                "system_mxid_localpart": "notices",
+                "system_mxid_display_name": "display name",
+                "system_mxid_avatar_url": "test/url",
```

Comment:
I'm unsure it is useful to provide defaults here, but I don't feel strongly! (I'm assuming without it we would go from `None` -> `"new display name"`, which probably still illustrates the code works?)

Reply:
yes, it will still illustrate the code works. I have removed the defaults.